notebooks/README.md

```diff
@@ -1,5 +1,20 @@
-## Cortex Certifai Tutorials 
-This directory contains example Jupyter notebooks illustrating how to use Certifai in a notebook environment.
+# Cortex Certifai Notebooks
+This directory contains *Jupyter notebooks* illustrating how to use Certifai in a notebook environment.
```

Comment:
Are we going to duplicate this information in the docs and here? Can't we just refer to docs for a description?  Or vice versa? @smichalski-cs

Reply:
I think we should keep the descriptions in the docs and I can link to the examples repo there

Reply:
Just link to the description page in the docs. I will link to the repo from the docs